catalog/edf-methanesat-ee/projects_edf-methanesat-ee_assets_public-preview_L3concentration.jsonnet

```diff
@@ -75,7 +75,7 @@ local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
       {
         name: 'terrain_height',
         description: 'Elevation with respect to WGS84 reference ellipsoid.',
-        'gee:units': units.kilometer,
+        'gee:units': units.meter,
```

Comment:
but the pixel values are in km

Reply:
Yes, you are right.  When I made this change I thought I had read in the code that was meters, but I should have just looked at the file itself.